### DISCUSSION TITLE
Target Rate - making persistent changes to target rate config

### DISCUSSION THREAD

**Discussion by baxgit** [BottlecapDave/HomeAssistant-OctopusEnergy#898] [Q&A]
Hi. Apologies if I've missed this in the docs, but I want to permanently change the configuration of an existing target rate sensor through a dashboard view - the scenario is that I have a number input helper on a dashboard that can be changed by my family which results in the relevant target rate config being amended (e.g. increase / decrease run hours, adjust start / end times etc). I can see that there is a 'service' which can create a temporary change to an existing target rate sensor, but can't see how to do make the change permanent. Any help is much appreciated. Thanks

**Comment by BottlecapDave** [comment]
Hello. You're not missing anything. There is currently no way of permanently changing the config outside of the integrations page. When the service was first introduced, there was no documented service for updating config permanently. When I get a chance, I'll see if this has changed.
